index.d.ts

```diff
@@ -17,11 +17,15 @@ import 'egg-schedule';
 import 'egg-static';
 import 'egg-jsonp';
 import 'egg-view';
+import EggCookies = require('egg-cookies');
 
 declare module 'egg' {
   // plain object
   type PlainObject<T = any> = { [key: string]: T };
 
+  // Remove specific property from the specific class
+  type RemoveSpecProp<Class,Prop> = Pick<Class,Exclude<keyof Class,Prop>>;
```

Comment:
change to

```typescript
type RemoveSpecProp<T, U> = Pick<T, Exclude<keyof T, U>>;
```

Its more standardized for ts, see https://www.typescriptlang.org/docs/handbook/advanced-types.html